app/views/posts/index.html.erb

```diff
@@ -0,0 +1,9 @@
+
+<% @posts.each do |post| %>
+<p class="ptitle"><%= post.title%></p>
+<p class="pauthor"><%= post.author%></p>
+<div class="go_postal">
+  <p class="pcontent"><%= post.post_content[0,700]%>...(click below for full post)</p>
+  <p><%= link_to"see more",post_path(post),:class => "a"%></p>
```

Comment:
I would avoid using class names like "a", which in your stylesheet could be confused for an anchor (`<a>`) tag.